tests/x11/chromium.pm

```diff
@@ -30,6 +30,7 @@ sub run {
     # Additional waiting to prevent unready address bar
     # https://progress.opensuse.org/issues/36304
     wait_still_screen(1);
+    assert_screen 'chromium-hightlighted-urlbar';
```

Comment:
Not only here.
We talked about adding it before each `type_string`.

There are two more cases:
```perl
    wait_screen_change { send_key 'ctrl-l' };
    type_string "https://html5test.opensuse.org\n";
# ...
    wait_screen_change { send_key 'ctrl-l' };
    type_string("https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg\n");
```

Reply:
done